pages/project/[id].js

```diff
@@ -76,51 +105,89 @@ export default function Task({
           <Textarea
             rows={10}
             maxLength="250"
-            placeholder="Write your note here .."
+            placeholder="take notes ..."
             name="note"
             defaultValue={selectedProject.notes}
             onChange={(event) => setCount(event.target.value.length)}
           ></Textarea>
           <NotesCounter>
             <Counter maxLength={250} counter={count} />
           </NotesCounter>
-          <SaveNoteButton type="submit">save note</SaveNoteButton>
+          <SaveNoteButton type="submit">save</SaveNoteButton>
         </NotesForm>
         <TaskForm onHandleAddTask={onHandleAddTask} projectid={id} />
+        <StyledPlusIcon />
       </Wrapper>
     </>
   );
 }
 
+const ErrorWrapper = styled.div`
+  position: absolute;
+  top: 10em;
+  left: 1em;
+  width: 85%;
+  height: 15%;
+  background-color: aliceblue;
+  border-radius: 10px;
+  margin: 1em;
+  padding: 1em;
+`;
+
+const ErrorMessage = styled.div`
+  font-size: 18px;
+  color: #696969;
+`;
+
+const StyledLink = styled(Link)`
+  position: absolute;
+  bottom: 1em;
+  border-radius: 20px;
+  padding: 0.5em;
+  background-color: #94c3dd;
+  box-shadow: 0 4px 14px 0 rgb(0 0 0 / 10%);
+`;
+
 const Header = styled.header`
   display: flex;
   flex-direction: column;
-  padding: 0.9em;
+  padding: 10px;
   align-items: center;
 `;
+
+const AppName = styled.h1`
+  position: absolute;
+  color: #94c3dd;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  top: 0em;
+  left: 0.7em;
+`;
+
 const BackButton = styled.div`
```

Comment:
As I told you before the `BackButton` is not a good name for a back link. Please change that.